### PR TITLE
Pagination screen reader tweak

### PR DESCRIPTION
**Pull request by lovemecomputer**
setting new screen reader text after convo with @dakahn .

also removing some code smell of exposing the button text as a prop—should not be configurable.

### CODE REVIEW

**Comment by lovemecomputer** [comment]
https://github.com/IBM/carbon-components/pull/2136 was merged while i was working on this, so i just made a new PR. files changed view still shows a good summary of the a11y tweaks here

File: src/components/PaginationV2/PaginationV2.js [hunk 1]
```diff
@@ -139,9 +129,7 @@ export default class PaginationV2 extends Component {
   };
 
   static defaultProps = {
-    backwardText: 'Backward',
     itemRangeText: (min, max, total) => `${min}-${max} of ${total} items`,
-    forwardText: 'Forward',
```
Comment (tw15egan) [inline src/components/PaginationV2/PaginationV2.js]: Shouldn't we be setting the new text as default, not as an unchangeable hardcoded value? Thinking of internationalization

Comment (lovemecomputer) [reply src/components/PaginationV2/PaginationV2.js]: @tw15egan oh wow yes true!!!

File: src/components/PaginationV2/PaginationV2.js [hunk 2]
```diff
@@ -338,32 +326,16 @@ export default class PaginationV2 extends Component {
           <button
             className={backButtonClasses}
             onClick={this.decrementPage}
-            aria-label={backwardText}
+            aria-label="previous page"
             disabled={backButtonDisabled}>
-            {componentsX ? (
-              <CaretLeft24 />
-            ) : (
-              <Icon
```
Comment (asudoh) [inline src/components/PaginationV2/PaginationV2.js]: `v9` need to keep the old icon?

Comment (lovemecomputer) [reply src/components/PaginationV2/PaginationV2.js]: @asudoh i think this whole code block is already inside of a `return componentsX ? ( … )` statement

**Comment by lovemecomputer** [comment]
i updated the story to match the default props for better screen reader text!

File: src/components/PaginationV2/PaginationV2.js (hunk 2 above)
Comment (asudoh) [reply src/components/PaginationV2/PaginationV2.js]: Got it - Thank you for clarifying @lovemecomputer!

**Pull request merged**